python/lance/tests/test_schema_evolution.py

```diff
@@ -68,13 +69,11 @@ def test_add_column_with_literal(tmp_path: Path):
     dataset = lance.dataset(base_dir)
     assert dataset.version["version"] == 1
     # Created within 10 seconds, in UDT
-    assert dataset.version[
-        "timestamp"
-    ] > datetime.datetime.utcnow() - datetime.timedelta(0, 10)
-    # Let Python auto convert timezones
-    assert dataset.version[
-               "timestamp"
-           ] > datetime.datetime.now() - datetime.timedelta(0, 10)
+    ts = dataset.version["timestamp"]
+    with pytest.raises(TypeError):
+        # can't compare offset-naive and offset-aware datetimes
+        assert ts > datetime.datetime.utcnow() - datetime.timedelta(0, 10)
```

Comment:
if the first assert raise exception, the second assert won't run?

Reply:
nvm. i thought the line 76 is also protected by `pytest.raise` .

Reply:
discussed. added newline for readability